src/data/firestore/attendees.ts

```
@@ -0,0 +1,46 @@
+import { db } from "@/config/firebase";
+import { AttendeeID, CamperAttendeeID, StaffAttendeeID, AdminAttendeeID, Attendee } from "@/types/sessionTypes";
```

ESLint

[error] 'CamperAttendeeID' is defined but never used.

(@typescript-eslint/no-unused-vars)